repos/system_upgrade/el7toel8/actors/checkbrltty/actor.py

```diff
@@ -0,0 +1,48 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.models import InstalledRedHatSignedRPM, BrlttyMigrationDecision
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
```

Comment:
Please mind imports order https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md#5-avoid--imports-and-sort-imports-alphabetically

Reply:
Fixed.